src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.Setup.csproj

```diff
@@ -200,5 +200,5 @@ folder InstallDir:\MSBuild\15.0\Bin\Roslyn\zh-Hant
     </ItemGroup>
   </Target>
 
-  <Import Project="..\..\..\NuGet\Microsoft.Net.Compilers\CompilerArtifacts.targets"/>
-</Project>
\ No newline at end of file
+  <Import Project="..\..\..\NuGet\Microsoft.Net.Compilers.Toolset\DesktopCompilerArtifacts.targets"/>
+</Project>
```

Comment:
Related to this PR: could you add an entry to https://github.com/dotnet/roslyn/wiki/NuGet-packages to list this new package with a one-line description of the purpose?

Reply:
Done.